pkg/workflow/manager.go

```diff
@@ -298,39 +356,150 @@ func (w *WorkflowManager) enqueueController(obj interface{}) {
 	w.queue.Add(key)
 }
 
+// enqueueAfter enqueue's a workflow after a given time.
```

Comment:
Enqueue's is 'bezittelijk', as enqueue does not own a workflow change to enqueues